### DISCUSSION TITLE
时间轴(TimeBar)支持默认单一时间点配置吗

### DISCUSSION THREAD

**Discussion by sanwanliu** [antvis/G6#3605] [Q&A]
![image](https://user-images.githubusercontent.com/23079982/161362631-e0b3d33f-bb08-4101-8ebc-e07efb56e8b6.png)

时间轴(TimeBar)支持默认单一时间点配置吗，我们不需要时间范围，一进页面就是默认的单一时间

**Comment by WangXang** [comment]
遇到同样的疑问？如何处理？

**Comment by WangXang** [comment]
找到一个不正规的方案：
我看源码里是这么干的：minShape、foregroundShape、minTextShape 全部hide
![image](https://user-images.githubusercontent.com/23691331/184057773-46189633-b628-4d9d-8649-9b91f14a4f75.png)
我们在timebar渲染完成后，直接获取这三个玩意，手动调用hide，在配置文件中配置隐藏”时间切换控件“(hideTimeTypeController:true),
最后配置slider居中
![image](https://user-images.githubusercontent.com/23691331/184058068-57d25fca-737d-4cba-a8f0-c8a2402bffb9.png)
